packages/backend/src/modules/update-monitor/UpdateNotifier.ts

```diff
@@ -277,15 +276,9 @@ export function generateTemplatizedStatus(): string {
   }[] = []
 
   for (const stack of stacks) {
-    const isFullyTemplatizedL2 = layer2s
-      .filter((l2) => l2.display.stack === stack)
-      .filter((l2) => !l2.isUpcoming && !l2.isArchived && !l2.isUnderReview)
-      .map((l2) => isDiscoveryDriven(l2))
-    const isFullyTemplatizedL3 = layer3s
-      .filter((l3) => l3.display.stack === stack)
-      .filter((l3) => !l3.isUpcoming && !l3.isArchived && !l3.isUnderReview)
-      .map((l3) => isDiscoveryDriven(l3))
-    const isFullyTemplatized = isFullyTemplatizedL2.concat(isFullyTemplatizedL3)
+    const isFullyTemplatized = scaling
+      .filter((p) => p.scalingInfo.stack === stack)
+      .map((p) => isDiscoveryDriven(p))
```

Comment:
What would be even better would be to have a discoveryInfo field on a project that would just tell you if it is disco-driven.